modules/add_person.rb

```diff
@@ -0,0 +1,21 @@
+def char_to_bool(char)
+  if char.downcase == 'y'
+    true
+  elsif char.downcase == 'n'
+    false
+  end
+end
+
+def add_student(age, name)
+  print 'Has parent permission? [Y/N]: '
+  $stdout.flush
+  par_perm = char_to_bool(gets.chomp)
+  Student.new(age, name, par_perm)
+end
```

Comment:
- [ ] At the moment select the answer to `Has parent permission? [Y/N]` question it returned an error, please fix this issue to be able to create a student 👇🏽 

```ruby
Has parent permission? [Y/N]: Y
/home/franclobo/microverse/school-library/person.rb:13:in `initialize': wrong number of arguments (given 3, expected 1..2) (ArgumentError)
        from /home/franclobo/microverse/school-library/student.rb:7:in `initialize'
        from /home/franclobo/microverse/school-library/modules/add_person.rb:13:in `new'
        from /home/franclobo/microverse/school-library/modules/add_person.rb:13:in `add_student'
        from /home/franclobo/microverse/school-library/app.rb:26:in `create_person'
        from main.rb:27:in `block in main'
        from main.rb:21:in `loop'
        from main.rb:21:in `main'
        from main.rb:38:in `<main>'
```

Reply:
Hello, @franclobo it's causing that issue because of the linter rule not allowing to use = and asking to use : on the default values. Should I remove the default value or disable the linter rule?